sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -1592,6 +1601,64 @@ public virtual VerifyResult VerifyData(SignatureAlgorithm algorithm, Stream data
             }
         }
 
+        /// <summary>
+        /// Creates an <see cref="RSA"/> implementation backed by this <see cref="CryptographyClient"/>.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> to cancel this operation.</param>
+        /// <returns>An <see cref="RSAKeyVault"/> implementation backed by this <see cref="CryptographyClient"/>.</returns>
+        /// <remarks>
+        /// The <see cref="CryptographyClient"/> will attempt to download the public key synchronously.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">This key is not of type <see cref="KeyType.Rsa"/> or <see cref="KeyType.RsaHsm"/>, or one or more key parameters are invalid.</exception>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        public virtual RSAKeyVault CreateRSA(CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _pipeline.CreateScope($"{nameof(CryptographyClient)}.{nameof(CreateRSA)}");
+            scope.AddAttribute("key", _keyId);
+            scope.Start();
+
+            try
+            {
+                Initialize(GetOperation, cancellationToken);
```

Comment:
Why not this, like the other methods?
```suggestion
                Initialize(nameof(CreateRSA), cancellationToken);
```

Reply:
Because I want to telemeter the `get` operation. Everything else is strictly client-side. That they called `CreateRSA` is captured just above.